src/biogeochem/CNFireLi2024Mod.F90

```diff
@@ -545,9 +538,12 @@ subroutine CNFireArea (this, bounds, num_soilc, filter_soilc, num_soilp, filter_
         c = filter_soilc(fc)
         g= col%gridcell(c)
         if(grc%latdeg(g) < cnfire_const%borealat )then
-           baf_peatf(c) = non_boreal_peatfire_c/secsphr*max(0._r8, &
-                min(1._r8,(4.0_r8-prec60_col(c)*secspday)/ &
-                4.0_r8))**2*peatf_lf(c)*(1._r8-fsat(c))
+            if ((trotr1_col(c)+trotr2_col(c))*col%wtgcell(c)<=0.8_r8.and.trotr1_col(c)+trotr2_col(c)>0.0_r8) then
```

Comment:
It would be good to add some comments about the code that changes to explain it. But, I don't want to hold anything up. So that could be done later, especially if an explanation would require working with Fang.

Reply:
Yeah, I'd need to work with Fang on that. I'll make sure to improve commenting as part of the fire refactor I have planned for later. Added this to my new issue #2714.